generate_endf80.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import shutil
+import sys
+import tarfile
+import tempfile
+import warnings
+import zipfile
+from multiprocessing import Pool
+from pathlib import Path
+from urllib.parse import urljoin
+
+import openmc.data
+
+from utils import download, process_neutron, process_thermal
+
+# Make sure Python version is sufficient
+assert sys.version_info >= (3, 6), "Python 3.6+ is required"
+
+description = """
+Generate ENDF/B-VIII.0 HDF5 library for use in OpenMC by first processing ENDF
+files using NJOY. The resulting library will contain incident neutron, incident
+photon, and thermal scattering data.
+
+"""
+
+
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                      argparse.RawDescriptionHelpFormatter):
+    pass
+
+
+parser = argparse.ArgumentParser(
+    description=description,
+    formatter_class=CustomFormatter
+)
+parser.add_argument('-d', '--destination', type=Path, default=Path('endfb80_hdf5'),
```

Comment:
Perhaps change the defualt dir to make it the consistance with others e.g. [endf-b7.1-hdf5](https://github.com/openmc-dev/data/blob/7c03af7da305305d9953a51475ae957f6bc7f948/generate_endf71.py#L35)


```suggestion
parser.add_argument('-d', '--destination', type=Path, default=Path('endf-b8.0-hdf5'),
```